docs/src/api.md

```diff
@@ -11,6 +11,7 @@ In this page we list the remaining API functions, which constitute the bulk of A
 
 ```@docs
 AgentBasedModel
+step!
```

Comment:
Hm, this will expand more than the Agents.jl-specific docstring. I think we need to write `step!(::AgentBasedModel)` instead.

Reply:
Perhaps it is worth adding an additional and first bullet point in the bullet list of `AgentBasedModel` saying something like "- `step!(model, args...)`: progress the model forwards in time. "

Reply:
Done. Not sure if the @ref in the bullet list will work, tough. How can I see a preview of the docs (online or local)?

Reply:
You cannot see a preview online unless you have push access to the repo.. To do it locally simply run the `docs/make
.jl` file.